src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -64,10 +65,34 @@ const ChannelImage = ({
   const isAnonymousCommenter = Boolean(anonPostNotificationUserInfo?.anon_user_info_emoji_code);
   const isAnonymousPostMaker = Boolean(postMaker?.anon_user_info_emoji_code);
   const isAnonymousOldPostMaker = Boolean(postMaker?.emoji_code);
+  const {anonProfileId} = useProfileHook();
 
   const renderChatMainImage = () => {
     const isAnonymousChannel = type === BaseChannelItemTypeProps.ANON_PM;
-    if (!isAnonymousChannel && isAnonymousPostMaker)
+    const memberChat = postMaker?.members?.find((item: any) => item.user_id !== anonProfileId);
+    const betterSocialMember = postMaker?.better_channel_member;
+
+    if (
+      isAnonymousChannel &&
+      isAnonymousPostMaker &&
+      betterSocialMember &&
+      betterSocialMember?.[memberChat?.user_id]?.anon_user_info_emoji_code &&
+      betterSocialMember?.[memberChat?.user_id]?.anon_user_info_color_code
+    ) {
+      return (
+        <ChannelAnonymousImage
+          anonPostNotificationUserInfo={{
+            anon_user_info_emoji_code:
+              betterSocialMember[memberChat?.user_id].anon_user_info_emoji_code,
+            anon_user_info_color_code:
+              betterSocialMember[memberChat?.user_id].anon_user_info_color_code
+          }}
+          imageStyle={styles.image}
+        />
+      );
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for rendering the main chat image has been significantly modified. It now includes checks for anonymous channels, post makers, and better social members. Make sure to test this new logic thoroughly to ensure it behaves as expected in all scenarios.
```diff
+ const {anonProfileId} = useProfileHook();
+ const renderChatMainImage = () => {
+   const isAnonymousChannel = type === BaseChannelItemTypeProps.ANON_PM;
+   const memberChat = postMaker?.members?.find((item: any) => item.user_id !== anonProfileId);
+   const betterSocialMember = postMaker?.better_channel_member;
+   if (
+     isAnonymousChannel &&
+     isAnonymousPostMaker &&
+     betterSocialMember &&
+     betterSocialMember?.[memberChat?.user_id]?.anon_user_info_emoji_code &&
+     betterSocialMember?.[memberChat?.user_id]?.anon_user_info_color_code
+   ) {
+     return (
+       <ChannelAnonymousImage
+         anonPostNotificationUserInfo={{
+           anon_user_info_emoji_code:
+             betterSocialMember[memberChat?.user_id].anon_user_info_emoji_code,
+           anon_user_info_color_code:
+             betterSocialMember[memberChat?.user_id].anon_user_info_color_code
+         }}
+         imageStyle={styles.image}
+       />
+     );
+   }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->